NxMapData.lua

```diff
@@ -1,4 +1,4 @@
-﻿---------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------
```

Comment:
I can't get rid of this change in the github UI, maybe someone can? I don't know what Git thinks has changed here. I copied the line multiple times in the editor and it somehow shows it as "changed"